Go/tests/syntax_test_go.go

```diff
@@ -5998,8 +5998,8 @@ func lang_embedding() {
     require.Equal(t, 1, testdb.QueryInt(env.testDb, `select count(*) from "schema.{{.site.table}}" order by {{.order}}`))
     //                                              ^ meta.string.go string.quoted.backtick.go punctuation.definition.string.begin.go
     //                                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.string.go meta.embedded.go source.sql.embedded.go
-    //                                                                            ^^^^^^^^^^^^^^^ meta.string.go meta.embedded.go source.sql.embedded.go meta.interpolation.go - string
-    //                                                                                           ^^^^^^^^^^^ meta.block.go meta.string.go meta.embedded.go source.sql.embedded.go
```

Comment:
might need to revisit this one - I disabled prototype inside quoted sql strings to prevent `#` from being scoped as a comment, but it had the side effect of disabling interpolation here... TBH I really think interpolation in SQL strings is a bad idea open to SQL injections, so I'd be okay with not highlighting it as a Go template here...